bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/utils/LatencyParser.java

```diff
@@ -24,9 +25,10 @@
  *
  * @author Andreas Hirsch - Initial contribution
  */
+@NonNullByDefault
 public class LatencyParser {
 
-    private static Pattern LATENCY_PATTERN = Pattern.compile(".*time=(.*) ?ms");
+    private static final Pattern latencyPattern = Pattern.compile(".*time=(.*) ?ms");
```

Comment:
I think SAT will complain about this casing now that the final modifier was?

```suggestion
    private static final Pattern LATENCY_PATTERN = Pattern.compile(".*time=(.*) ?ms");
```